packages/documentation/src/stories/foundation/accessibility/regulation.docs.mdx

```diff
@@ -0,0 +1,17 @@
+import { Meta, Source, Canvas } from '@storybook/blocks';
+import * as RegulationStories from './regulation.stories';
+
+<Meta of={RegulationStories} />
+
+# Accessibility
+
+<figure>
+  <blockquote class="blockquote" lang="de">
+    <p>Die Postdienste müssen so angeboten werden, dass Menschen mit Behinderungen
+sie in qualitativer, quantitativer und wirtschaftlicher Hinsicht unter vergleichbaren
+Bedingungen wie Menschen ohne Behinderungen beanspruchen können.</p>
+  </blockquote>
+  <figcaption class="blockquote-footer">
+    <cite>§14.7 <a href="https://www.fedlex.admin.ch/eli/cc/2012/585/de#art_14">Swiss postal law</a></cite>
```

Comment:
So the language can be choosen by the page itself, depending on the browser language.
```suggestion
    <cite>§14.7 <a href="https://www.fedlex.admin.ch/eli/cc/2012/585">Swiss postal law</a></cite>
```